packages/url_launcher/url_launcher_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.0-nullsafety
```

Comment:
do we need to do web as well? cc @ditman

Reply:
I'm thankful this is migrated, however integration_tests aren't currently running.

Reply:
circling back, you guys discussed offline to revert `url_launcher_web`. I'm fine with that. Thanks for the effort @mvanbeusekom!

Reply:
@ditman Maybe I can revert it back in this PR and open a second one just for the url_launcher_web package as draft and have it ready to go when all dependencies get fixed?

Reply:
@mvanbeusekom that sounds great, that way all the effort doesn't go to waste! :)